### ISSUE TITLE
import keras hd5 model (inception v3)

### ISSUE THREAD

**Issue by benman1** [deeplearning4j/deeplearning4j#4676]
#### Issue Description

I am trying to import a model, the inception v3 model, from hd5 format as downloaded here: https://github.com/tomthetrainer/KerasWorkshop/releases. With the stable version 0.9.1 from maven central I ran into problems with reading the hd5 (the same as [this issue](https://github.com/deeplearning4j/deeplearning4j/issues/3520)) I found there seemed to have been a few changes around reading hd5 exported from keras since 0.9.1 ([release notes](https://github.com/deeplearning4j/deeplearning4j/pull/3831)), so I am using the sonatype snapshot 0.9.2-SNAPSHOT. However now, I am getting another issue: 

```java
Exception in thread "main" org.deeplearning4j.nn.modelimport.keras.exceptions.UnsupportedKerasConfigurationException: Unsupported batch normalization axis 1DL4J currently supports axis=3 only. Consider reshaping your input.. Please file an issue at http://github.com/deeplearning4j/deeplearning4j/issues.
	at org.deeplearning4j.nn.modelimport.keras.layers.normalization.KerasBatchNormalization.<init>(KerasBatchNormalization.java:87)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasLayerUtils.getKerasLayerFromConfig(KerasLayerUtils.java:224)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.prepareLayers(KerasModel.java:190)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:158)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:92)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelBuilder.buildModel(KerasModelBuilder.java:123)
	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasModelAndWeights(KerasModelImport.java:124)
	at example.importModel.main(importModel.java:33)
```

My line is just this:
```java
        ComputationGraph model = KerasModelImport.importKerasModelAndWeights(
                "src/main/resources/inception_v3_complete.h5"
        );

```

**Comment by maxpumperla** [comment]
@benman1 well, the issue is precisely what the error message says, you want batchnorm along dimension 1, while import only supports axis 3.

If I recall correctly, we have a new batch norm implementation in nd4j that's more flexible than the current one. I'll see what I can do about this.

**Comment by tomthetrainer** [comment]
@maxpumperla That is almost certainly a h5 file from  keras 1.x export. 

Do we intend to have Keras 1.x support in current and future versions? 

If not you can ignore this as far as Model Import is concerned and I can make it a documentation issue and put up versions of inception exported from keras 2.x? 

@benman1 There are saved keras models of inception using keras 2 here. 
https://github.com/fchollet/deep-learning-models/releases

If you just want the quickest route to inception in dl4j I have a saved inception here, 

https://github.com/SkymindIO/Demos-for-sales-marketting/releases

The code to import it is also in that repo, but it should be straightforward, ModelSerializer.LoadModel(?)

Something like that.

**Comment by maxpumperla** [comment]
@tomthetrainer yes, we intend to have full backward compatibility. so that's perfectly fine. 👍

**Comment by maxpumperla** [comment]
@benman1 I relaxed the conditions for batch norm axes again. be aware that this might lead to batch norm leading to unexpected results in weird edge cases (that you prob don't want).
